File: VB/Solution41.Module/BO.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

Namespace Solution41.Module
	<DefaultClassOptions> _
	Public Class Client
		Inherits Person
		Public Sub New(ByVal s As Session)
			MyBase.New(s)
		End Sub
	End Class

	<DefaultClassOptions> _
	Public Class SaleTask
		Inherits Task
		Public Sub New(ByVal s As Session)
			MyBase.New(s)
		End Sub
	End Class
End Namespace

File: CS/Solution41.Module/BO.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace Solution41.Module {
    [DefaultClassOptions]
    public class Client : Person {
        public Client(Session s) : base(s) { }
    }

    [DefaultClassOptions]
    public class SaleTask : Task {
        public SaleTask(Session s) : base(s) { }
    }
}
